src/main/java/ivorius/reccomplex/block/TileEntityBlockScript.java

```diff
@@ -68,13 +68,14 @@ public NBTTagCompound writeToNBT(NBTTagCompound nbtTagCompound)
     @Override
     public void generate(StructureSpawnContext context, WorldScriptMulti.InstanceData instanceData)
     {
-        script.generate(context, instanceData, pos);
+    	if (spawnTriggerable || (redstoneTriggerable && redstoneTriggered))
```

Comment:
Why do you start testing for spawnTriggerable (or redstoneTriggerable too) here? The spawnTriggerable check in prepareInstanceData takes care of determining when it should spawn.